deploy/helm/sumologic/values.yaml

```diff
@@ -295,274 +295,66 @@ prometheus-operator:
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.operator.rule
-        writeRelabelConfigs:
-        - action: keep
-          regex: ':node_memory_utilisation:|:node_memory_MemFreeCachedBuffers_bytes:sum|:node_memory_MemTotal_bytes:sum|node:node_memory_bytes_available:sum|node:node_memory_bytes_total:sum|node:node_memory_utilisation:ratio|node:cluster_memory_utilisation:ratio|:node_memory_swap_io_bytes:sum_rate|node:node_memory_utilisation:|node:node_memory_utilisation_2:|node:node_memory_swap_io_bytes:sum_rate'
-          sourceLabels: [__name__]
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.operator.rule
-        writeRelabelConfigs:
-        - action: keep
-          regex: ':node_disk_utilisation:avg_irate|node:node_disk_utilisation:avg_irate|:node_disk_saturation:avg_irate|node:node_disk_saturation:avg_irate'
-          sourceLabels: [__name__]
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.operator.rule
-        writeRelabelConfigs:
-        - action: keep
-          regex: 'node:node_filesystem_usage:|node:node_filesystem_avail:'
-          sourceLabels: [__name__]
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.operator.rule
-        writeRelabelConfigs:
-        - action: keep
-          regex: ':node_net_utilisation:sum_irate|node:node_net_utilisation:sum_irate|:node_net_saturation:sum_irate|node:node_net_saturation:sum_irate'
-          sourceLabels: [__name__]
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.operator.rule
-        writeRelabelConfigs:
-        - action: keep
-          regex: 'node:node_inodes_total:|node:node_inodes_free:'
-          sourceLabels: [__name__]
-      # up metrics
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics
-        writeRelabelConfigs:
-        - action: keep
-          regex: up
-          sourceLabels: [__name__]
-      # prometheus remote write metrics
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics
-        writeRelabelConfigs:
-        - action: keep
-          regex: 'prometheus_remote_storage_.*'
-          sourceLabels: [__name__]
-      # fluentd metrics
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics
-        writeRelabelConfigs:
-          - action: keep
-            regex: 'fluentd_.*'
-            sourceLabels: [__name__]
-      # fluent-bit metrics
-      - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics
-        writeRelabelConfigs:
-          - action: keep
-            regex: 'fluentbit.*'
-            sourceLabels: [__name__]
+        # kube state metrics
+        - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.state
```

Comment:
Will you need to change the plugin as tags are changed?

Reply:
Don't believe so, but need @samjsong to confirm.  In my testing, all things worked like I expected, data volume diffs were accounted for.  I do not believe we depend on on the tag in the plugins as we use a label to handle the datapoint stuff.

Reply:
we do depend on the tag in the plugins, we no longer do the retag based on job to make it clearer to the user where the metrics were going. In this case the tags are correct since we check for `<match prometheus.metrics.state**>` etc.